src/emotion.d.ts

```diff
@@ -0,0 +1,9 @@
+import '@emotion/react';
+import theme from './theme';
+
+type ThemeType = typeof theme;
+
+declare module '@emotion/react' {
```

Comment:
Note that `@emotion/react` **does** have types, so we're _extending_ here vs declaring.